springwolf-bindings/springwolf-googlepubsub-binding/src/main/java/io/github/springwolf/bindings/googlepubsub/annotations/channels/GooglePubsubAsyncMessageStoragePolicy.java

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: Apache-2.0
-package io.github.springwolf.bindings.googlepubsub.annotations;
+package io.github.springwolf.bindings.googlepubsub.annotations.channels;
 
 import java.lang.annotation.ElementType;
```

Comment:
There is a typo in the name of the annotation, it should be GooglePub**S**ubAsyncMessageStoragePolicy. It would be great if you could change it while you are at it.